labgrid/driver/power/digipower.py

```diff
@@ -38,13 +38,13 @@ def power_get(host, port, index):
     r.raise_for_status()
     states = {"0": False, "1": True}
     ports = {
-        "1": 10,
-        "2": 11,
-        "3": 12,
-        "4": 13,
-        "5": 14,
-        "6": 15,
-        "7": 16,
-        "8": 17
+        1: 10,
+        2: 11,
+        3: 12,
+        4: 13,
+        5: 14,
+        6: 15,
+        7: 16,
+        8: 17
     }
```

Comment:
This is basically the same as above, but comma-separated. So something like this instead of states/ports/return?
```
state = r.text.split(',')[9+index]
return bool(int(state, 2))
```